### PR TITLE
Chorus command

### LINKED ISSUE #727: Chorus Integration (Clone Hero)
> Clone Hero is a free rhythm game, which can be played with any 5 or 6 button guitar controller, game controllers, or just your standard computer keyboard.

**[Chorus](https://chorus.fightthe.pw/)** ([GitHub](https://github.com/Paturages/chorus#api-documentation)) is a user organised repository of user provided Clone Hero songs. The website lets you search and download charts for the game.

### Chorus API documentation
https://github.com/Paturages/chorus#api-documentation

### Assets (from the website)
**Bass** https://chorus.fightthe.pw/assets/images/instruments/bass.png
**Drums** https://chorus.fightthe.pw/assets/images/instruments/drums.svg
**Guitar** https://chorus.fightthe.pw/assets/images/instruments/guitar.png
**Keys** https://chorus.fightthe.pw/assets/images/instruments/keys.png
**Chorus Logo** https://chorus.fightthe.pw/assets/images/logo.png
**Chorus Icon** https://chorus.fightthe.pw/assets/images/favicon_128.png

### PR DESCRIPTION
**Pull request by pedrofracassi**
Closes #727 

- [x] Search for songs on chorus
- [x] Display chart name
- [x] Display chart artist
- [x] Download links just like the website
- [x] Display instruments
- [x] Display difficulties per instrument **in order** (EMHX)
- [x] Note per second average
- [x] Note per second rating
- [x] Source links
- [x] Display warnings
- [x] Filter out `null` or `undefined` lines and blocks

### CODE REVIEW

**Comment by pedrofracassi** [comment]
image

**Comment by pedrofracassi** [comment]
image

File: src/commands/games/chorus.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+const { SearchCommand, SwitchbladeEmbed, MiscUtils, Constants } = require('../../')
+
+const color = '#2C3E50'
```
Comment (perronosaurio) [inline src/commands/games/chorus.js]: add this in constants

Comment (pedrofracassi) [reply src/commands/games/chorus.js]: Done!

**Comment by pedrofracassi** [comment]
RFR!

**Pull request merged**